src/odemis/driver/test/xt_client_test.py

```diff
@@ -712,6 +711,72 @@ def test_apply_autofocus(self):
         autofocus_state = self.microscope.is_autofocusing(channel)
         self.assertEqual(autofocus_state, False)
 
+    def test_get_mpp_orientation(self):
+        """
+        Simple test, checks if no error occur during calling the method and the return value.
```

Comment:
Rephrase: `Test getting the multiprobe pattern orientation.`

Do rephrase also all the other test doc strings the same way. Then they are all consistent with the rest.